src/frontend/webpack.config.development.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
   plugins: [...commonConfig.plugins, new ForkTsCheckerNotifierWebpackPlugin()],
   devServer: {
     historyApiFallback: true,
+    allowedHosts: 'all',
     client: {
```

Comment:
```suggestion
    headers: { 'Access-Control-Allow-Origin': '*' },
    client: {
```

Reply:
If this solution runs from altinn3local.no, this could make it better at reporting errors. I've also added a `crossorigin` property on the script tag for our test apps which improves this even more (and made Cypress able to catch detailed errors).

Reply:
Nice, thanks for the suggestion 🙂